assets/css/_route_pill.scss

```diff
@@ -10,6 +9,18 @@
   line-height: 1.5rem;
   font-weight: 700;
   font-family: $font-family-route-pill;
+
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
```

Comment:
question(non-blocking): why the switch to flex?

Reply:
Would `line-height: 1.5` work? That way it would auto-adjust based on the font size

https://developer.mozilla.org/en-US/docs/Web/CSS/line-height#number

Reply:
The height to font-size ratio is not consistent for both versions of this button. So font-size:height for the default version is 1:1.5 while font-size:height for the large format is 1.25:2.25 or 1:1.8

Reply:
Ahhh - I had thought it was 1.5 : 2.25 and 1 : 1.5, which would have been consistent.

I'll back out of this conversation then, since I don't have an opinion on this (I would probably have gravitated towards `flex` too, fwiw)